tests/link/pytorch/test_elemwise.py

```diff
@@ -150,3 +153,33 @@ def test_cast():
         fgraph, [np.arange(6, dtype="float32").reshape(2, 3)]
     )
     assert res.dtype == torch.int32
+
+
+def test_vmap_elemwise():
+    from pytensor.link.pytorch.dispatch.basic import pytorch_funcify
+
+    class TestOp(ScalarOp):
```

Comment:
I'm not sure if this should be defined at the module level or function level. If you want me to move it out lmk

Reply:
doesn't matter. Usually I put it inside the test if it's not reused